src/main/java/com/hubspot/jinjava/interpret/JinjavaInterpreter.java

```diff
@@ -498,6 +498,13 @@ public int getPosition() {
   }
 
   public void addError(TemplateError templateError) {
+
+    // fix line numbers not matching up with source template
+    if (!context.getCurrentPathStack().isEmpty()) {
```

Comment:
I believe this is needed for `addAllErrors` as well. I'm also wary of any place that might be doing `getErrors().add()` instead of this method.